Makefile

```diff
@@ -65,7 +65,7 @@ nsxiv: $(OBJS)
 
 $(OBJS): Makefile nsxiv.h config.h commands.h
 options.o: version.h
-window.o: icon/data.h
+window.o: icon/data.h utf8.h
```

Comment:
This should be conditioned on HAVE_LIBFONTS being true

Reply:
I know. But didn't think it was worth it, as that would mean more `_0/1` macro trickery.

Reply:
Fair enough.